CHANGELOG.md

```diff
@@ -4,6 +4,11 @@ by [**SWR Audio Lab**](https://lab.swr.de/)
 
 ## Changelog
 
+- 2022-12-19 - v1.1.0
+  - feat: add helpers package (with `sleep` function)
+  - feat: add numbers package (with `toReadable` function)
+  - feat: add `capitalize` and `pluralize` functions in strings package
+  - chore: move `isEven` from strings to numbers package (BREAKING CHANGE)
```

Comment:
I think long term we should match the Changelog-style with our other projects

Reply:
what do you mean? please specify or suggest changes